### ISSUE TITLE
timeStep has no effect in Kronos 1.5.1

### ISSUE THREAD

**Issue by Nantris** [dubert/react-kronos#53]
Thanks very much for the great work @dubert.

I am trying to use the timeStep property and I have tried passing it in in many ways to no effect.

```
timeStep={15}
timeStep="15"
```
and
```
const kronosOptions = {
    timeStep: 15
}

<kronosComponent {...kronosOptions } />
```

None of these produced any effect? Thanks for any ideas you may have.

**Comment by felixakiragreen** [comment]
Thanks! Fixed in 1.5.2

**Comment by Nantris** [comment]
Thanks so much!
